docs/OPENSHIFT.md

```diff
@@ -0,0 +1,186 @@
+# LogDNA Agent on OpenShift
+
+The agent has been tested on OpenShift 4.4, but should be compatible with any OpenShift cluster packaged with Kubernetes version 1.9 or greater.
```

Comment:
let's just recommend on OpenShift 4.4+ — because the customer doesn't care if you've tested or not, just a guarantee that it works with X version

Reply:
Sounds good, also this should be `4.6` and not `4.4`